platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -367,6 +367,11 @@ pre code {
   color: $navigation_task_color;
 }
 
+.nav-group-name > .small-heading,
+.nav-group-task-link {
+  display: block;
```

Comment:
Oh, I forgot to mention: this also extends the hit target for the links in the navigation list to the right, so that you can click the link without having the mouse cursor directly over the link text.